### ISSUE TITLE
Add an automated way to get tweets un-retweeted

### ISSUE THREAD

**Issue by squeek502** [plorry/VegAssist#24]
I haven't looked into the possibilities for how to do this, but it might be some useful functionality to have:
- If the Twitter API can notify the bot when they have been blocked, then we could go through and un-retweet any tweets from the user that blocked it.
- We could track @replies to the bot and have some specific message that will get things un-retweeted.

I think the first would be best, as the second would require more effort on the users part and would be more difficult to communicate. A simple thing like "block this bot to stop your tweets from being retweeted" seems like a straightforward way to go about it.

We should also make sure that we already can't retweet tweets from users that have us blocked. Not sure if `statuses/filter` API filters those tweets out for us or not.

**Comment by squeek502** [comment]
Looks like the first implementation isn't possible. You can [only retrieve a list of users that _you_ are blocking](https://dev.twitter.com/rest/reference/get/blocks/ids), not a list of users that have blocked you.

**Comment by squeek502** [comment]
> We should also make sure that we already can't retweet tweets from users that have us blocked. Not sure if `statuses/filter` API filters those tweets out for us or not.

This doesn't seem possible either. The API doesn't return different results for someone that has you blocked. `statuses/user_timeline` and `statuses/show/:id` both return normal results.

https://twittercommunity.com/t/getting-blockers-and-spammers-by-rest-api/11723/6

**Comment by plorry** [comment]
@squeek502 I guess that makes sense.
I was thinking that a message about unRTing would go in the bio, since you can see that information on a hover-over, without needing to click through to the profile.
"Tweet 'unRT me' to me to unretweet" could fit in bio (or something like that that reads better :\ ). Then we implement the second method of auto-unretweeting.

**Comment by squeek502** [comment]
Yeah, however, it's a bit weird that you can't check if specific users have you blocked using the REST API, since [you can see that information when you visit in the profile of a user that has you blocked on the website](https://support.twitter.com/articles/20172060?lang=en).

But, yeah, looks like "tweet this to me to un-retweet" is probably our best bet.
